esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -145,43 +172,27 @@ String bluetooth_device_name = ""; // "mLRS BT"; // Bluetooth device name, "" re
 
 #include "mlrs-wireless-bridge-boards.h"
 
-
-//-------------------------------------------------------
-// Includes
-//-------------------------------------------------------
-
-#if ESP_ARDUINO_VERSION < ESP_ARDUINO_VERSION_VAL(3, 0, 0)
-    #error Version of your ESP Arduino Core below 3.0.0 !
-#elif ESP_ARDUINO_VERSION < ESP_ARDUINO_VERSION_VAL(3, 0, 4)
-    #warning Consider upgrading your ESP Arduino Core ! // warnings may not be displayed in console !
-#endif
-
 #ifdef GPIO0_IO
 #define USE_AT_MODE
 #endif
 
-#include <WiFi.h>
-#include "esp_mac.h"
-#if (WIRELESS_PROTOCOL != 2) // not UDPCl
-// for some reason checking
-// #if defined(CONFIG_BT_ENABLED) && defined(CONFIG_BLUEDROID_ENABLED)
-// does not work here. Also checking e.g. PLATFORM_ESP32_C3 seems not to work. 
-// This sucks. So we don't try to be nice but let the compiler work it out.
-#if defined USE_AT_MODE || (WIRELESS_PROTOCOL == 3) // for AT commands we require BT be available
-  #define USE_WIRELESS_PROTOCOL_BLUETOOTH
-  #include <BluetoothSerial.h>
-#endif
-#endif
-
-
 //-------------------------------------------------------
 // Internals
 //-------------------------------------------------------
 
-#if (WIRELESS_PROTOCOL != 2) // WiFi TCP, UDP, BT
+#if (WIRELESS_PROTOCOL != 2) // not UDPCl
+
+#if (defined USE_AT_MODE || (WIRELESS_PROTOCOL == 3)) && ! defined ESP8266 // for AT commands we require BT be available except 8266
+  #define USE_WIRELESS_PROTOCOL_BLUETOOTH
+  #include <BluetoothSerial.h>
+#endif
 
 // WiFi TCP, UDP
+#ifdef ESP8266 // could eliminate this and use broadcast for first message
```

Comment:
pl remove the comment
I think it is not esp8266 specific, and it also is poiting into the future

Reply:
I didn't say the comment was esp8266 specific.  The comment is specific to the ifdef itself.  Using broadcast for the first message would remove the need for the ifdef. The last octet would be set to 255 for both esp32 and esp8266.  I intend do to that at some point, but I didn't want to do it in this PR.  The comment is to remind us that it should probably be broadcast.

Reply:
is understood, but the placement of the comment alludes to it's esp8266 specific
so leave it, I will change or remove after merge